[Doc] Webhooks documentation tweaks (#624)
huggingface/hub-docs: docs/hub/webhooks.md

Webhooks are a foundation for MLOps related features. You can use them to auto-convert models, build community bots, or build CI/CD for your models, datasets, and Spaces.

> They allow you to listen for new changes on specific repos or to all repos belonging to particular users/organizations (not just your repos, but any repo).

Reviewer: ```suggestion
They allow you to listen for new changes on specific repos or to _all_ repos belonging to particular users/organizations.
```

You can create new webhooks and edit existing ones in your webhooks [settings](https://huggingface.co/settings/webhooks):

![Settings of an individual webhook](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/webhook-settings.png)

> Webhooks can watch for repos updates, Pull requests, discussions, and new comments. If you need to host your webhook, it's of course possible to use a Space to host it.

Reviewer 2: for me, the previous version "It's even possible to create a Space to react to your webhooks!" was clearer

Author: yes, feel free to revert + merge then (let's wait for @coyotte508 review maybe)

Reviewer: i'm split!

Author: ok will revert that sentence.

cc @osanseviero fyi too

> Webhooks can watch for repos updates, Pull requests, discussions, and new comments. If you need to host your webhook, it's of course possible to use a Space to host it.

Author: ```suggestion
Webhooks can watch for repos updates, Pull requests, discussions, and new comments. It's even possible to create a Space to react to your webhooks!
```
